website/_sass/color_schemes/eps-dark.scss

```diff
@@ -1,3 +1,14 @@
 @import "./color_schemes/dark";
 @import "_variables";
 @import "_common";
+
+// Swap `$blue-000` and `$blue-300`, mainly for callouts. This is done by
+// default for red, but not for other colors.
+$blue-000: #183385;
+$blue-300: #2c84fa;
+
+$grey-dk-000: $grey-lt-000;
+$grey-dk-100: $grey-lt-100;
+$grey-dk-200: $grey-lt-200;
+$grey-dk-250: $grey-lt-200;
+$grey-dk-300: $grey-lt-300;
```

Comment:
Just checking: only 250 having an "off' mapping is intended?

Reply:
I think it is based on the color's [defined](https://just-the-docs.github.io/just-the-docs/docs/utilities/color/#dark-greys) by `just-the-docs`. I believe the idea was for the `summary` boxes to be light instead of dark in a dark theme, as otherwise it does not stand out as much.

Reply:
Wow :smile: (Maybe worth a comment; not sure.)

Reply:
Makes sense. Added a comment :+1:.